### PR TITLE
Add WithAutoRollbackOnFailure ibgu pkg function

### PR DESCRIPTION
**Pull request by mcornea**

### CODE REVIEW

File: pkg/ibgu/ibgu.go [hunk 1]
```diff
@@ -113,6 +113,31 @@ func (builder *IbguBuilder) WithClusterLabelSelectors(labels map[string]string)
 	return builder
 }
 
+// WithAutoRollbackOnFailure appends the AutoRollbackOnFailure to the ibuSpec.
+func (builder *IbguBuilder) WithAutoRollbackOnFailure(initMonitorTimeout int) *IbguBuilder {
+	if valid, _ := builder.validate(); !valid {
+		return builder
+	}
+
+	glog.V(100).Infof("Creating IBGU with %d AutoRollbackOnFailure InitMonitorTimeout", initMonitorTimeout)
```
[the remark this answers is not in the document]
Comment (mcornea) [reply pkg/ibgu/ibgu.go]: tnx, updated.

File: pkg/ibgu/ibgu.go [hunk 2]
```diff
@@ -113,6 +113,32 @@ func (builder *IbguBuilder) WithClusterLabelSelectors(labels map[string]string)
 	return builder
 }
 
+// WithAutoRollbackOnFailure appends the AutoRollbackOnFailure to the ibuSpec.
```
Comment (achuzhoy) [inline pkg/ibgu/ibgu.go:116]: I'd add the InitMonitorTimeout related stuff to this comment too, but ...

**Review by achuzhoy — APPROVED** [review]

File: pkg/ibgu/ibgu.go (hunk 2 above)
Comment (mcornea) [reply pkg/ibgu/ibgu.go]: sure, added.

**Review by achuzhoy — APPROVED** [review]

**Review by klaskosk — APPROVED** [review]
/lgtm

**Pull request merged**